### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by DavidWChen**
## Description
Updated README
Changed error in Account Creation email
Change [0] to .first()

## Pls do 
- [x] Tested and working
- [x] Unit tests

## Test plan
`make test`
Postman
screen shot 2017-12-05 at 4 10 46 pm

### CODE REVIEW

File: src/users/views.py [hunk 1]
```diff
@@ -99,14 +99,15 @@ def post(self, request):
...
         to_email = Email(new_user.email)
         subject = "Pick a Brain with PickMyBruin!"
         content = Content("text/html", 
                 "Thank you for joining BQuest! We are happy you are here! Click the link below to verify that you are a UCLA student, and create your profile. \n" +
                 "You will be receiving emails from us, so make sure to update your address on your profile, if you prefer to use another email.\n"+
-                url+ new_profile.verification_code+
-                "If you have any questions, feel free to contact us on bquest.ucla@gmail.com.")
+                url+ new_profile.verification_code+ 
+                "\nIf you have any questions, feel free to contact us on bquest.ucla@gmail.com.\n"
+                )
```
Comment (marktai) [inline src/users/views.py]: can you have the the close paren on the same indentation level as the open.

Also, for blocks of text, I use 

'/n'.join([
  line1, 
  line2, 
  line3, 
])

File: src/users/models.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ class Mentor(models.Model):
     major = models.ForeignKey(Major, null=True, on_delete=models.SET_NULL)
     bio = models.CharField(max_length=5000, null=False, blank=True, default='')
     active = models.BooleanField(default=True)
-    gpa = models.DecimalField(default=0.00, max_digits=4, decimal_places=2)
+    gpa = models.DecimalField(blank=True, default=0.00, max_digits=4, decimal_places=2)
```
Comment (marktai) [inline src/users/models.py]: any reason why?

Comment (DavidWChen) [reply src/users/models.py]: Ram was getting an error when using mentors/me/, and the error was the same as when I forgot to allow classes and clubs to be blank. I couldn't replicate the error, so I just put it in to be safe.

Comment (marktai) [reply src/users/models.py]: you want null=True. but I highly suspect Ram didn't run a migration

Comment (DavidWChen) [reply src/users/models.py]: So take it out, or switch it to null=True?

Comment (marktai) [reply src/users/models.py]: I think you can take it out. We already have a default

Comment (marktai) [reply src/users/models.py]: we shouldn't add random code to fix a problem if we're not sure it even fixes it

**Pull request merged**